test/arrays.js

```diff
@@ -0,0 +1,117 @@
+var assert = require('assert');
+var diff = require('../index.js');
+
+describe('Array Diff', function() {
+  it('both children edit same key to same array values', function() {
```

Comment:
Correct, similar to "both children edit same key to same values" you wrote in literals.js